WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -37,20 +317,20 @@ extension WordPressAppDelegate {
         })
     }
 
-    @objc func configureCrashLogging() {
+    func configureCrashLogging() {
```

Comment:
@frosty:

This method a little out of date with develop – now we just call `WPCrashLogging.start()` inside of it. I've encapsulated the send-or-don't send logic in that class so that it's easier to implement in other projects.

Now that we're not doing the Obj-C => Swift thing, IMHO we could just eliminate the `configureCrashLogging` method and just call `WPCrashLogging.start()` directly?